### DISCUSSION TITLE
Watch command in Synology Docker

### DISCUSSION THREAD

**Discussion by vicegold** [Taxel/PlexTraktSync#1252] [Q&A] [answered]
Hi,

I'm trying to run PlexTraktSync via Synology GUI, but in the `Watch` state. Has anybody managed to do that? I'm using the `twolaw/PlexTraktSync` package from @twolaw.

Seems like the Synology GUI doesn't let me change the startup command, so I'm not quite sure how this could work.

Cheers

**Comment by simonc56** [comment, marked as answer]
You should use the [official package](https://github.com/Taxel/PlexTraktSync/pkgs/container/plextraktsync) to get support here.

**Comment by scoiattolo49** [comment]
You can use this image (lscr.io/linuxserver-labs/plextraktsync):
https://github.com/linuxserver-labs/docker-plextraktsync

The package is already set up to run in watch state. Sync is done periodically using a cnotab file: you can change it if you want it less often (especially if you have a big library)
